### DISCUSSION TITLE
Version not supported yet 1.21.3

### DISCUSSION THREAD

**Discussion by Akirakato1** [PrismarineJS/mineflayer#3513] [Q&A] [answered]
The server I'd like to connect to is 1.21.3 but mineflayer seems to only support up to 1.21.1. Is this because minecraft-protocols only supports up to 1.21.1 too? Is there a work around or will 1.21.3 be supported officially? I've tried to spoof my 1.21.1 mineflayer bot client as 1.21.3 or make the server appear to be 1.21.1 but both with no success, perhaps I've made mistakes in the places to edit code. Any help for work around for version mismatch is appreaciated.

**Comment by rom1504** [comment, marked as answer]
https://github.com/PrismarineJS/mineflayer/pull/3489 and dependencies PRs needs to be finished
